application/listeners/posthelpers_test.go

```diff
@@ -112,23 +112,27 @@ func (ms *ModelSuite) TestSendNotificationRequestFromStatus() {
 		domain.ErrLogger.SetOutput(os.Stderr)
 	}()
 
+	var getT = notifications.GetEmailTemplate
```

Comment:
Depending on whether you change the scope of the other `getT`, you could combine them into one.